src/app/api/comment/[id]/route.tsx

```
@@ -0,0 +1,64 @@
+            content: "This is a comment for project 1.",
+            createdAt: "2025-05-13T11:13:59.194Z",
+        },
+        {
+            id: 9007199254740992,
+            projectId: "1",
+            userName: "Jane Smith",
+            content: "Another comment for project 1.",
+            createdAt: "2025-05-14T12:15:45.123Z",
+        },
+        {
+            id: 9007199254740993,
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 9007199254740992

(lint/correctness/noPrecisionLoss)